haml/_footer.html.haml

```diff
@@ -7,39 +7,9 @@
       %span.cads-footer__feedback-icon.cads-icon_external-link
   %nav.cads-grid-container{"aria-label": t("cads.footer.navigation_title") }
     .cads-grid-row
-      - if links["column_one"]
-        .cads-grid-col-md-3
-          %h2.cads-footer__section-title= links["column_one"]["title"]
-          - if links["column_one"]["links"]
-            %ul.cads-footer__links
-              - links["column_one"]["links"].each do |link|
-                %li.cads-footer__link
-                  %a.cads-footer__hyperlink{ href: link["url"] }= link["label"]
-      - if links["column_two"]
-        .cads-grid-col-md-3
-          %h2.cads-footer__section-title= links["column_two"]["title"]
-          - if links["column_two"]["links"]
-            %ul.cads-footer__links
-              - links["column_two"]["links"].each do |link|
-                %li.cads-footer__link
-                  %a.cads-footer__hyperlink{ href: link["url"] }= link["label"]
-      - if links["column_three"]
-        .cads-grid-col-md-3
-          %h2.cads-footer__section-title= links["column_three"]["title"]
-          - if links["column_three"]["links"]
-            %ul.cads-footer__links
-              - links["column_three"]["links"].each do |link|
-                %li.cads-footer__link
-                  %a.cads-footer__hyperlink{ href: link["url"] }= link["label"]
-      - if links["column_four"]
-        .cads-grid-col-md-3
-          %h2.cads-footer__section-title= links["column_four"]["title"]
-          - if links["column_four"]["links"]
-            %ul.cads-footer__links
-              - links["column_four"]["links"].each do |link|
-                %li.cads-footer__link
-                  %a.cads-footer__hyperlink{ href: link["url"] }= link["label"]
-
+      - %w[column_one column_two column_three column_four].each do |column|
+        - if links[column]
+          = render partial: "footer_links", locals: { links: links[column] }
```

Comment:
Do you mind inlining this partial? Currently using `render` _within_ a partial in design-system severely limits what the `view_path` can be on the Rails side. This problem would go away if/when we move to publishing a rails engine, but for now it's a restriction. See "Allows more granular prepend_view_path
" in https://github.com/citizensadvice/design-system/pull/458 for more context. The loop makes sense though.